### PR TITLE
Upgrade to latest solc 0.5.8

### PR DESCRIPTION
**Pull request by elenadimitrova**
Also due to https://github.com/dapphub/ds-note/issues/4 removed `DSToken` from tests and instead use instances of our own `Token`

### CODE REVIEW

File: truffle.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ module.exports = {
     }
   },
   mocha: {
-    reporter: "mocha-circleci-reporter",
+    reporter: "eth-gas-reporter",
```
Comment (area) [inline truffle.js]: Mistakenly commited?

Comment (elenadimitrova) [reply truffle.js]: yes :(

**Comment by elenadimitrova** [comment]
Not a bad idea with the `UnlockedToken` although I would rather be explicit with the Token model we have unless you mind terribly?

**Comment by area** [comment]
No, was just wondering. It'd have pros and cons, so happy to leave as-is.

**Pull request merged**